File: android/FastingTimerPro/app/src/main/java/com/fastingtimerpro/app/ui/theme/AppColors.kt
package com.fastingtimerpro.app.ui.theme

import androidx.compose.ui.graphics.Color

object AppColors {

    // BACKGROUND
    val baseBackground = Color(0xFF0B1220)
    val gradientTop = Color(0xFF0F1A2E)
    val gradientBottom = Color(0xFF070B14)

    // ACCENT COLORS
    val primaryAccent = Color(0xFF4DA3FF)
    val secondaryAccent = Color(0xFF6FD3FF)

    // TEXT
    val primaryText = Color.White
    val secondaryText = Color(0xFFA9B4C7)
    val mutedText = Color(0xFF7C8698)

    // GLASS / CARDS
    val cardBackground = Color.White.copy(alpha = 0.08f)
    val cardBorder = Color.White.copy(alpha = 0.18f)

    // BUTTONS
    val primaryButtonBackground = Color(0xFF4DA3FF).copy(alpha = 0.25f)
    val secondaryButtonBackground = Color.White.copy(alpha = 0.10f)
    val destructiveButtonBackground = Color(0xFFFF5A5A).copy(alpha = 0.35f)

    // HIGHLIGHTS
    val brightRed = Color(0xFFFF5A5A)
}

File: ios/FastingTimerPro/Sources/AppColors.swift
import SwiftUI

enum AppColors {

    // MARK: - Background
    static let baseBackground = Color(hex: 0x0B1220)
    static let gradientTop = Color(hex: 0x0F1A2E)
    static let gradientBottom = Color(hex: 0x070B14)

    // MARK: - Accent
    static let primaryAccent = Color(hex: 0x4DA3FF)
    static let secondaryAccent = Color(hex: 0x6FD3FF)

    // MARK: - Text
    static let primaryText = Color.white
    static let secondaryText = Color(hex: 0xA9B4C7)
    static let mutedText = Color(hex: 0x7C8698)

    // MARK: - Glass / Cards
    static let cardBackground = Color.white.opacity(0.08)
    static let cardBorder = Color.white.opacity(0.18)

    // MARK: - Buttons
    static let primaryButtonBackground = Color(hex: 0x4DA3FF).opacity(0.25)
    static let secondaryButtonBackground = Color.white.opacity(0.10)
    static let destructiveButtonBackground = Color(red: 1.0, green: 0.35, blue: 0.35).opacity(0.35)

    // MARK: - Highlights
    static let brightRed = Color(hex: 0xFF5A5A)
}

extension Color {
    init(hex: UInt, alpha: Double = 1.0) {
        self.init(
            .sRGB,
            red: Double((hex >> 16) & 0xFF) / 255.0,
            green: Double((hex >> 8) & 0xFF) / 255.0,
            blue: Double(hex & 0xFF) / 255.0,
            opacity: alpha
        )
    }
}
